### ISSUE TITLE
webpack throws WARNING in configuration

### ISSUE THREAD

**Issue by svenlindstroem** [sveltejs/sapper#1012]
**Describe the bug**
Building with sapper webpack throws warning when running npm run build. 

**Logs**
WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/

**To Reproduce**
npm run build

* If possible, we recommend creating a small repo that illustrates the problem.
* Reproductions should be small, self-contained, correct examples – http://sscce.org.

**Expected behavior**
No warnings

**Information about your Sapper Installation:**
macOS High Sierra
sapper webpack
node 12.9.1
sapper 0.27.0
svelte 3.0.0

**Severity**
annoying

**Comment by JefferyHus** [comment]
Have you tried to do `set node_env=production && npm run build`

**Comment by svenlindstroem** [comment]
Yes thanks, I have got it running. Still, should be in the readme.  Or adding a .env file.

**Comment by tvanc** [comment]
On Windows I had to install `cross-env` as a dev dependency:
```npm i -D cross-env```

and change `scripts.build` in `package.json` to this:
```node_modules/.bin/cross-env NODE_ENV=production \"npm run build:tailwind && sapper build\"```

Anything else either got the warning the OP described or caused Webpack to throw this exception.
```> Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.mode should be one of these:
   "development" | "production" | "none"
   -> Enable production optimizations or development hints.```
